src/openforms/appointments/core.py

```diff
@@ -116,5 +115,4 @@ def book_for_submission(submission: Submission) -> str:
         logevent.appointment_register_failure(appointment_info, plugin, e)
         raise AppointmentRegistrationFailed("Unable to create appointment") from e
 
-    cancel_previous_submission_appointment(submission)
```

Comment:
considering this whole function is new-style appointments only, I'm not so sure about this line being removed :thinking:

Reply:
I am not sure that I understand your point. I removed it because we do not have to deal with an extra function in order to cancel an appointment (we have `CancelAppointmentView`) right?And basically I have removed the whole function from `appointments/utils.py`.

Reply:
I was wrong - this was incorrectly added when the new-style appointments was created and the removal is correct